### ISSUE TITLE
Website Feedback

### ISSUE THREAD

**Issue by ecnahc515** [coreos/bugs#465]
I have a few suggestions that I'd like to see personally on the CoreOS website:

1) The chronological ordering of releases at https://coreos.com/releases/ is actually fairly hard to follow. 

As a deployer, I often want to know how a particular channel has progressed over time. It would be great if we could track what versions of CoreOS have gone into stable at any point in time, rather than only knowing what the 'current' stable is. If a customer has disabled auto-updates and we've gone through 2 versions of CoreOS stable, it will be very difficult to determine what the version which went into stable before the most recent.

I would suggest something similar to the EC2 page, where we have tabs for each channel, each tab having a different list of AMIs for each channel. (https://coreos.com/os/docs/latest/booting-on-ec2.html). We could have some kind of tabs per channel, each listing all releases for that channel.

2) A 'detail' page for a given release.

I didn't know until today that it was possible to get a full list of installed packages on CoreOS. Apparently we have these pages: http://alpha.release.core-os.net/amd64-usr/current/coreos_production_image_packages.txt but I doubt many people are aware of this. Having links to the images, and text files available in the http://alpha.release.core-os.net/amd64-usr/$VERSION/ on a per-release detail page would be awesome.

Then users could go to our release page, click the release they're interested in, and find all links to the images they want, or a link to the list of packages on the given release. This would be super helpful to audit changes between releases prior to upgrading your production deployments.

**Comment by ecnahc515** [comment]
CC @crawford @robszumski 

None of these are time sensitive, or critical, but I wanted to write these down before I forgot, and make sure the thoughts are being relayed to the other teams.

**Comment by jonboulle** [comment]
+1, I've wanted both of these things before

**Comment by crawford** [comment]
For number 1, the ordering can be found on https://alpha.release.core-os.net/amd64-usr/ (or beta or stable). We just need to show that nicely.

**Comment by robszumski** [comment]
https://coreos.com/releases/ now contains a link to https://coreos.com/releases/releases.json

Additionally, there are:

https://coreos.com/dist/aws/aws-alpha.json
https://coreos.com/dist/aws/aws-beta.json
https://coreos.com/dist/aws/aws-stable.json
